setup.py

```diff
@@ -97,9 +97,8 @@ def checkProgramIsInstalled(self, program, args, where_to_download,
                        "matplotlib >= 2.0.0",
                        "intervaltree >= 2.1.0",
                        "pyBigWig >=0.3.7",
-                       "future >= 0.16.0",
-                       "hicmatrix",
-                       "pysam",
+                       "hicexplorer >= 2.1.1",
```

Comment:
please adopt also numpy to the one in requirements

Reply:
I have removed as many as i could find of python2 stuffs while ago. I would have assumed it is already part of the current release.

Reply:
Yes, this is LeilyR commit but it was because the develop branch was behind master by one comit...
@LeilyR, in fact, I do not see why we require hicexplorer >= 2.1.1. For me, the former were more close to what is actually needed, no?
@bgruening hicexplorer or hicmatrix require numpy so I do not know if we should specify here...

Reply:
I am not sure if hicexplorer is needed at all. I think only hicmatrix should be there. I can check.

Reply:
What I mean is that `"numpy >= 1.12.1"` is different from `numpy >= 1.16`. So setup.py is different from requirements.py.

@joachimwolff can comment more on what is currently used for hicex.

Reply:
@LeilyR, yeah, I mean I would put hicmatrix, maybe future... In the conda recipe there is no pygenometracks:
```
requirements:
  host:
    - python >=3.6
    - pip
  run:
    - python >=3.6
    - numpy >=1.16
    - matplotlib >=3.0
    - intervaltree >=2.1.0
    - pybigwig >=0.3.4
    - future >=0.17.0
    - hicmatrix >=9
    - pysam
```